runtime/src/commonMain/kotlin/app/cash/sqldelight/Transacter.kt

```diff
@@ -178,7 +180,9 @@ private class TransactionWrapper<R>(
 /**
  * A transaction-aware [SqlDriver] wrapper which can begin a [Transaction] on the current connection.
  */
-abstract class TransacterImpl(private val driver: SqlDriver) : Transacter {
+abstract class TransacterImpl<StatementType : SqlPreparedStatement, CursorType : SqlCursor>(
+  private val driver: SqlDriver<StatementType, CursorType>,
```

Comment:
likewise here, this driver is only ever used in the file to do the notification APIs which dont care about the type, so I think you can just do `SqlDriver<SqlPreparedStatement, SqlCursor>` or `SqlDriver<out SqlPreparedStatement, out SqlCursor>` or some shit like that

Reply:
Done!